x-pack/filebeat/input/azureblobstorage/mock/data_random.go

```diff
@@ -104,7 +104,7 @@ func generateRandomBlob() []byte {
 }
 
 func createRandomData() MyData {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Why are we seeding with time? This makes failures non-reproducible, so prefer a defined seed. If you want to have fuzzing, then we should have a Fuzz test.

Reply:
will fix the seed value